CONTRIBUTING.md

```diff
@@ -1,6 +1,6 @@
 # Contributing to Dep
 
-Dep is an open source project.
+`dep` is an open source project.
```

Comment:
Do we want to change other instances of `Dep` to `dep` in this doc?, like on line 1?